pkg/operator/sync.go

```diff
@@ -691,6 +705,64 @@ func (optr *Operator) syncMachineConfigPools(config *renderConfig) error {
 	return nil
 }
 
+// we need to mimic this
+func (optr *Operator) syncMachineConfigNodes(config *renderConfig) error {
+	fg, err := optr.fgAccessor.CurrentFeatureGates()
+	if err != nil {
+		klog.Errorf("Could not get fg: %w", err)
+		return err
+	}
+	if !fg.Enabled(configv1.FeatureGateMachineConfigNodes) {
+		return nil
+	}
+	nodes, err := optr.nodeLister.List(labels.Everything())
+	if err != nil {
+		return err
+	}
+	for _, node := range nodes {
+		var pool string
+		var ok bool
+		if pool, ok = node.Labels["node-role.kubernetes.io/worker"]; ok {
+			pool = "worker"
```

Comment:
Hmm, why are these hard-coded? What about custom pools?

Reply:
True, I was thinking for TechPreview we only track master/worker. I can add custom though if you think it makes sense

Reply:
keeping with worker and master for tech preview.